manifests/cli/config.pp

```diff
@@ -24,8 +24,10 @@
   validate_string($ssh_private_key_content)
 
   # required by PuppetX::Jenkins::Provider::Clihelper base
-  package { 'retries':
-    provider => 'gem',
+  if ! defined(Package['retries']) {
```

Comment:
@jhoblitt it is not an ideal situation given the different distributions. If we use 'defined' it should at least be backwards compatible. Then again given that this affects the experimental types only, we may just remove it altogether and make it easier for everyone going forward.